src/main/webapp/style.css

```diff
@@ -1,9 +1,169 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/*--------------
+Imported Fonts
+---------------*/
+@import url('https://fonts.googleapis.com/css?family=Montserrat:100,200,300,400,500,600,700,800,900');
+@import url('https://fonts.googleapis.com/css?family=Sriracha');
+
+/*--------------
+Site Name
+---------------*/
+#site-name {
+  color: #00bcd4;
+  font-family: 'Sriracha', Helvetica, Arial, sans-serif;
+  font-size: 40px;
+  font-weight: 400;
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties): Use shorthand properties where possible.

Here and below you can simply use the `font` property to set these values